### ISSUE TITLE
SAMI2 instrumentation broke the docs

### ISSUE THREAD

**Issue by aburrell** [pysat/pysatModels#56]
**Describe the bug**
Introducing the SAMI2 model instrument to `supported_models` broke the ability of sphinx to use autodoc

**To Reproduce**
1. Go to 'docs'
2. Run 'make html'
3. See the compilation warnings

**Expected behavior**
The html should compile without warnings

**Comment by jklenzing** [comment]
Having issues with `m2r`, so I can't build these locally

**Comment by aburrell** [comment]
It's looking for something called "coords", but I can't find an inappropriate coords reference anywhere.

**Comment by jklenzing** [comment]
I've got my setup fixed.  Not getting a warning related to coords, but it's cranky that two web links use the same text.  Pushed updates, let me know if those work for you.

Also getting errors about the contributing doc.  Should we just convert that over to rst in the main directory?  Then we only have to deal with one format style when we import.

**Comment by aburrell** [comment]
I added commits for the link names and contributing to the doc PR draft.  I'll checkout and merge your PR and see if readthedocs is ok with the SAMI2 instrument...

**Comment by aburrell** [comment]
Ok, you can see the problem in the readthedocs log at https://readthedocs.org/api/v2/build/11506400.txt:

```
WARNING: autodoc: failed to import module 'models.sami2py_sami2' from module 'pysatModels'; the following exception was raised:
cannot import name 'general' from 'pysat.instruments.methods' (/home/docs/checkouts/readthedocs.org/user_builds/pysatmodels/envs/develop/lib/python3.7/site-packages/pysat/instruments/methods/__init__.py)
WARNING: autodoc: failed to import module 'models.ucar_tiegcm' from module 'pysatModels'; the following exception was raised:
cannot import name 'general' from 'pysat.instruments.methods' (/home/docs/checkouts/readthedocs.org/user_builds/pysatmodels/envs/develop/lib/python3.7/site-packages/pysat/instruments/methods/__init__.py)
WARNING: autodoc: failed to import module 'utils.match' from module 'pysatModels'; the following exception was raised:
cannot import name 'general' from 'pysat.instruments.methods' (/home/docs/checkouts/readthedocs.org/user_builds/pysatmodels/envs/develop/lib/python3.7/site-packages/pysat/instruments/methods/__init__.py)
WARNING: autodoc: failed to import module 'utils.extract' from module 'pysatModels'; the following exception was raised:
cannot import name 'general' from 'pysat.instruments.methods' (/home/docs/checkouts/readthedocs.org/user_builds/pysatmodels/envs/develop/lib/python3.7/site-packages/pysat/instruments/methods/__init__.py)
WARNING: autodoc: failed to import module 'utils.compare' from module 'pysatModels'; the following exception was raised:
cannot import name 'general' from 'pysat.instruments.methods' (/home/docs/checkouts/readthedocs.org/user_builds/pysatmodels/envs/develop/lib/python3.7/site-packages/pysat/instruments/methods/__init__.py)
looking for now-outdated files... none found
```

I am guessing it has something to do with pysat :/

**Comment by jklenzing** [comment]
I think I see.  readthedocs is probably installing from pip, which means the general methods do not exist yet.  It does exist in the develop branch, which means that this should be fixed once pysat 2.2 is released.

**Comment by aburrell** [comment]
Will keep this open until the pysat 2.2 release and see if it resolved.
